ios/FluentUI/Other Cells/ActivityIndicatorCell.swift

```diff
@@ -74,7 +74,10 @@ open class ActivityIndicatorCell: UITableViewCell, TokenizedControlInternal {
 
     open override func willMove(toWindow newWindow: UIWindow?) {
         super.willMove(toWindow: newWindow)
-        tokenSet.update(fluentTheme)
+		guard let newWindow else {
```

Comment:
I'm sorry if all these aren't actual spacing issues but they look off on my end haha

Reply:
Nope it was very wrong, thank you!